benchmark/benchmark_lagrangian_particle_tracking.jl

```diff
@@ -33,7 +33,7 @@ end
 
 # Benchmark parameters
 
-Architectures = has_cuda() ? [CPU, GPU] : [CPU]
+Architectures = CUDA.functional() ? [CPU, GPU] : [CPU]
```

Comment:
This same exact line appears a dozen times... perhaps time to implement a utility for it...